tutorials/en_us/JARS/OPTIMIZATION.md

```diff
@@ -0,0 +1,19 @@
+# Intro
+
+Here you will learn how to optimize you Minecraft server to get better performance.
+
+## Optimizing your .jar specific configurations to YOUR NEEDS
+
+Every .jar comes with its own set of configurations you can tweak to suit your needs, and keep in mind that, YOUR NEEDS. You probably wont find as effective the config of a minigames server in a survival server. You should read every config, understand it, and choose if you need to use it or not. This is extremely important in survival servers as some config values can ruin your mob spawning rates or destroy your players custom redstone circuits.
```

Comment:
First sentence doesn't make grammatical sense.
> Every .jar comes with its own set of configurations you can tweak to suit your needs, and keep in mind that, YOUR NEEDS.

Possibly
`Every .jar has its own set of configurations that you can tweak to suit your specific needs`